sdk/storage/Azure.Storage.Blobs/src/AssemblyInfo.cs

```diff
@@ -15,3 +15,15 @@
     "012ea67d2479751a0b8c087a4185541b851bd8b16f8d91b840e51b1cb0ba6fe647997e57429265" +
     "e85ef62d565db50a69ae1647d54d7bd855e4db3d8a91510e5bcbd0edfbbecaa20a7bd9ae74593d" +
     "aa7b11b4")]
+[assembly: InternalsVisibleTo("Azure.Storage.Files.DataLake, PublicKey=" +
```

Comment:
Can I ask why we need internal access? It is generally a bad idea to couple these libraries like this. It will force use to always ship them together to ensure we don't break the internal api usage as well as means customers will always have to use the matching versions of these libraries as well to avoid breaking.

Reply:
We might not, we'll see later during the implementation.  The intention is to only be dependent on the public blob interfaces.

Reply:
Yes it would be best if we can depend only on the public apis.